src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -137,15 +143,39 @@ public CommandBase moveTicks(double ticks, double speed) {
                 CommandScheduler.getInstance().requiring(this).cancel();
             }
         }, () -> {
-            return Math.abs(liftEncoder.getDistance()) >= ticks;
+            return Math.abs(liftEncoder.getDistance()) >= Math.abs(0.10);
         }, this);
-        cmd.setName("Move Ticks");
+        cmd.setName("Move Disengage");
         return cmd.withTimeout(0.2);
     }
 
+    public CommandBase moveTicks(double ticks, double speed) {
```

Comment:
What is the distance of a "tick"? Please document.

Reply:
resolvified